internal/cmd/flags.go

```diff
@@ -41,6 +41,22 @@ var (
 		fs.StringVar(p, PflagSignatureFormat.Name, defaultSignatureFormat, PflagSignatureFormat.Usage)
 	}
 
+	PflagID = &pflag.Flag{
+		Name:  "id",
+		Usage: "key id (required if --plugin is set). This is mutually exclusive with the --key flag",
+	}
+	SetPflagID = func(fs *pflag.FlagSet, p *string) {
+		fs.StringVar(p, PflagID.Name, "", PflagID.Usage)
+	}
+
+	PflagPlugin = &pflag.Flag{
+		Name:  "plugin",
+		Usage: "signing plugin name. This is mutually exclusive with the --key flag",
+	}
+	SetPflagPlugin = func(fs *pflag.FlagSet, p *string) {
+		fs.StringVar(p, PflagPlugin.Name, "", PflagPlugin.Usage)
+	}
+
 	PflagTimestamp = &pflag.Flag{
```

Comment:
nit: Can we please remove timestamp flag as we will not be supporting in rc3?

cc: @shizhMSFT @patrickzheng200 @yizha1

Reply:
@patrickzheng200  @shizhMSFT I didn't see `timestamp` flag in the command line. Is it the flag that we should have removed it earlier?

Reply:
This is a dead code not referenced anywhere.